src/styles/BookItem.css

```diff
@@ -1,36 +1,169 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;300;400;500;600;700;800;900&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
+}
+
 .containerRender {
   display: flex;
   justify-content: space-between;
   height: 160px;
   width: 90%;
   margin: 20px auto;
+  background-color: #fff;
+  border-radius: 5px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 500;
 }
```

Comment:
- [ ] Great job so far. However, looking at your design carefully and comparing it with that of the given design I noticed a few missing details that when added to your design will help prove your attention to detail and also make your design look more professional. To rectify these missing details kindly consider the below suggestion to help you do just that: 
  - Kindly make sure that the card aligns with the nav text. If you look at your design carefully you will notice that your brand name(`bookstore`) is not on the same line as that of the starting point of your cards. To have a better understanding of what I mean I have included a screenshot of your design below and use red lines to mark where your design doesn't match that of the given design.
  - As I mentioned above kindly make sure your `profile` icon is also aligned with the ending point of the card.
  
  
<table>
<tr>
<th align=“middle”>original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="864" alt="Screenshot 2023-08-24 at 17 14 24" src="https://github.com/Bhekisile/Bookstore/assets/107192076/4593ea11-e99a-4a09-89ed-fa2308353b8f">
</td>
<td>
<img width="1439" alt="Screenshot 2023-08-24 at 21 42 26" src="https://github.com/Bhekisile/Bookstore/assets/107192076/ae4f2575-cd7a-4188-92fc-aace9fa945dc">
</td>
</tr>
</table>